engines/dradis-api/app/controllers/dradis/ce/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,42 @@
+module Dradis::CE::API
+  module V1
+    class CommentsController < Dradis::CE::API::V1::ProjectScopedController
+      before_action :set_comment, only: [:show, :update, :destroy]
+      before_action :set_commentable, only: [:index, :create]
+
+      def index
+        @comments = @commentable.comments
+      end
+
+      def show; end
+
+      def create
+      end
+
+      def update
+      end
+
+      def destroy
+      end
+
+      private
+
+      def set_commentable
```

Comment:
I had this idea to make this less dependent on the commendable type:
https://github.com/dradis/dradis-ce/pull/385/files#diff-6cead7405362cf0e4214e6479ed22755R44

Reply:
we're basing our design in Basecamp, and I believe this `if/elsif/elsif` would make DHH cringe.

I'm not sure if there is a better solution, but a quick search on StackOverflow yielded this:

https://stackoverflow.com/a/14959484